app/Traits/Eloquent/Remover.php

```
@@ -132,30 +98,368 @@ public function removeAllTheEntriesMediaFoldersLocal($projectRef): void
+
+                // Check if this is S3 or local storage
+                if ($config['driver'] === 's3') {
+                    $diskRoot = config('filesystems.disks.' . $driver . '.root').'/';
+                    $s3Client = $this->createS3Client($config);
+                    $bucket = $config['bucket'];
+                    $prefix = $diskRoot . $projectRef;
+                    $deletedCount = $this->deleteOneBatchByPrefixS3($s3Client, $bucket, $prefix, $remainingCapacity);
+                } else {
+                    // Handle local storage using Laravel Storage facade
+                    $deletedCount = $this->deleteOneBatchByPrefixLocal($driver, $projectRef, $remainingCapacity);
+                }
```

PHPMD (2.15.0)

The method removeMediaChunk uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (Clean Code Rules)

(ElseExpression)